tests/BUILD

```diff
@@ -42,6 +42,17 @@ checkstyle_test(
     size = "small",
 )
 
+py_test(
+    name = "test_concurrent",
+    srcs = [
+        "integration/test_concurrent.py",
+    ],
+    deps = [
+        "//:client_python",
+    ],
+    python_version = "PY3"
+)
+
```

Comment:
This integration test is a reproducible scenario for the error fixed in this PR.